atest/testdata/standard_libraries/operating_system/get_file.robot

```diff
@@ -113,15 +113,25 @@ Get Binary File returns bytes as-is
 Grep File
     [Template]    Grep And Check File
     ${EMPTY}    foo\nbar\nfoo bar\n\nA Foo
+    ${EMPTY}    foo\nbar\nfoo bar\n\nA Foo    regexp=${True}
     foo         foo\nfoo bar
+    foo         foo\nfoo bar    regexp=${True}
     foo?        foo bar
+    foo.        foo bar    regexp=${True}
     ?foo        ${EMPTY}
+    .foo        ${EMPTY}    regexp=${True}
     ?oo         foo\nfoo bar\nA Foo
+    .+oo         foo\nfoo bar\nA Foo    regexp=${True}
     [Ff]oo      foo\nfoo bar\nA Foo
+    .*[Ff]oo      foo\nfoo bar\nA Foo    regexp=${True}
     f*a         foo bar
+    f.*a         foo bar    regexp=${True}
     ?           foo\nbar\nfoo bar\nA Foo
+    .           foo\nbar\nfoo bar\nA Foo    regexp=${True}
     ????        foo bar\nA Foo
+    ....        foo bar\nA Foo    regexp=${True}
     foo bar     foo bar
+    foo\\sbar     foo bar    regexp=${True}
```

Comment:
1. I think it would be better to add separate `Get File with regexp` test. Then it needs to be added to matching file on the `atest/robot` side as well.
2. No need to use `${True}`, just `True` will work. I generally use both and also variants like `true` and even `yes` if such a flag is needed multiple times. No need for separate tests, though.

Reply:
The tests are now split up.

I do not know if I do these comments in the right way though. How do I get my changes into the PR?